classes/Http/Form/Form.php

```diff
@@ -13,12 +13,12 @@
 
 abstract class Form
 {
-    protected $_options;
-    protected $_messages;
-    protected $_cleanData;
-    protected $_taintedData;
-    protected $_purifier;
-    protected $_fieldList = [];
+    protected $options;
+    protected $messages;
+    protected $cleanData;
+    protected $taintedData;
+    protected $purifier;
+    protected $fieldList = [];
```

Comment:
These changes are potential BC breaks, if anyone else extends `Form`

Reply:
I think that's fine and arguably even a non-issue.